.github/workflows/ci.yml

```diff
@@ -137,6 +137,16 @@ jobs:
     if: github.event_name == 'pull_request'
     steps:
     - uses: actions/checkout@v4
+    - name: Get short commit SHA
+      id: short-sha
+      run: echo "sha=$(git rev-parse --short $GITHUB_SHA)" >> $GITHUB_OUTPUT
```

Comment:
there is something going on with this `$GITHUB_SHA`, it's not the commit that we can see on the branch. Could you maybe change/fix it with following infos? 

* https://stackoverflow.com/questions/68061051/get-commit-sha-in-github-actions
* https://github.com/orgs/community/discussions/26325

For the rest, the wheel installed and ran fine in my local env!

Reply:
Done. Please review changes @MyPyDavid.